### PR TITLE
Choim date gap fill

### PR DESCRIPTION
**Pull request by choim-sci**
@covesturtevant, 

This is date_gap_filler module pushed as neon-is-date-gap-fill in GCD.  Thanks

### CODE REVIEW

File: pipe/tchain/tchain_date_gap_filler.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 pipeline:
   name: tchain_date_gap_filler
 transform:
-  image_pull_secrets:
-  - battelleecology-quay-read-all-pull-secret
-  image: quay.io/battelleecology/date_gap_filler:c0ce8112
+  # image_pull_secrets:
+  # - battelleecology-quay-read-all-pull-secret
+  image: us-central1-docker.pkg.dev/neon-shared-service/neonscience/neon-is-date-gap-fill:v03.01.01
```
Comment (covesturtevant) [inline pipe/tchain/tchain_date_gap_filler.yaml:7]: Please use single digits for semantic version. If there is no existing semantic version in the existing quay.io repo, please start with v1.0.0

**Review by covesturtevant — CHANGES_REQUESTED** [review]
There are several more pipeline specifications that reference the quay.io image for the date_gap_filler. Please find and replace those instances with the GCP image. Otherwise looks good, thanks.

**Review by covesturtevant — CHANGES_REQUESTED** [review]
Looks good except for workflow for DEV_errored_datums_reader

**Comment by choim-sci** [comment]
@covesturtevant, 
Reverted.

**Review by covesturtevant — APPROVED** [review]
Looks great. Thanks.

**Pull request merged**